cmd/zetae2e/local/local.go

```diff
@@ -286,8 +286,10 @@ func localE2ETest(cmd *cobra.Command, _ []string) {
 	}
 	if testAdmin {
 		eg.Go(adminTestRoutine(conf, deployerRunner, verbose,
+			e2etests.TestZetaWithdrawName,
```

Comment:
why is this `TestZetaWithdrawName` required here?

Reply:
Nice catch, this is a line I added for testing purposes, I forgot to remove

Reply:
9297e265